wrap/gtwrap/matlab_wrapper/mixins.py

```diff
@@ -0,0 +1,222 @@
+"""Mixins for reducing the amount of boilerplate in the main wrapper class."""
```

Comment:
maybe we could define how we are using the term "mix-in" here?

Reply:
Mixins are a standard programming feature (we call them traits in C++). As such, it should be self-explanatory.

https://en.wikipedia.org/wiki/Mixin
https://stackoverflow.com/questions/533631/what-is-a-mixin-and-why-are-they-useful